kwok/README.md

```diff
@@ -47,12 +47,54 @@ spec:
 EOF
 ```
 
+## Specifying Instance Types
```

Comment:
Would also be nice if we could just generate the existing hard-coded instance types and put that in the filesystem, so that it always takes in a file, but you don't have to go fishing for how you might want to define it.  The only complaint there would be if the number of zones changed, or became variable, a static file would need some extra propagation work to succeed.

Reply:
Oh, sure, that should be easy to do.

Reply:
@njtran Ok, I created a `kwok/tools/gen_instances.go` file and moved the code that previously generated the static/hardcoded list in here.  `make gen_instances` will re-run and update the auto-generated list, and the helm charts will create the configmap and wire it all up together.

You can see these changes in the second commit of the stack.  I tested it out locally and it still seems to work.